_authors/osci_2023/prabhask.markdown

```diff
@@ -0,0 +1,7 @@
+---
+name: Prabhas Kurapati
+short_name: prabhask
+github: prabhask5
+---
+
+**<Prabhas Kurapati>** I've never broken a bone
```

Comment:
While this is a great personal fact, maybe just a brief description would be better.

Reply:
I see, thanks for the feedback! I submitted a new commit with that change.

Reply:
Thanks for your feedback! I committed a new description, thanks!